site/components/footer/index.js

```diff
@@ -20,31 +21,43 @@ import mapSVG from './SVG/map.svg';
 
 const cx = classnames.bind(styles);
 
+const trackAnalytics = title => () => ReactGA.event({
+  category: 'FooterNavigation',
+  action: title,
+  label: 'click',
+});
```

Comment:
Navigating actually broke the tracking :S I'll leave it as is for now and i'll bug it. For now, this is a big improvement. It's also blocking another PR, so getting this through ASAP would be great